Chap_API_Server.tex

```diff
@@ -1699,7 +1693,8 @@ \subsection{\code{PMIx_server_collect_inventory}}
 \argin{cbdata}{Data to be passed to the callback function (memory reference)}
 \end{arglist}
 
-Returns \refconst{PMIX_SUCCESS} or a negative value corresponding to a PMIx error constant. In the event the function returns an error, the \refarg{cbfunc} will not be called.
+\returnsimple
+In the event the function returns an error, the \refarg{cbfunc} will not be called.
```

Comment:
Revision Exception:

```suggestion
\returnsimplenb
```

 * Convert to `returnsimplenb`
 * Remove redundant sentence now that we use `returnsimplenb`